packages/fast-components-class-name-contracts-msft/src/call-to-action/index.ts

```diff
@@ -0,0 +1,15 @@
+import { IHypertextClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * The class name contract for the call to action component
+ */
+export interface IMSFTCallToActionClassNameContract extends IHypertextClassNameContract {
```

Comment:
There is no base MSFT component, so I don't think we need to preface this with MSFT.

Reply:
Sure, I suppose so.